docs/docs/sample-scripts/teams/create-team-from-group.md

```diff
@@ -0,0 +1,53 @@
+# Create a Microsoft Team with channels from a Microsoft 365 Group
+
+Inspired by: [Patrick Lamber](https://www.nubo.eu/Provision-A-Team-With-CLI-For-Microsoft-365/)
+
+A sample script which creates a Microsoft 365 Group, associates a logo to it and some members. Afterward, it teamyfies the Group and creates two public channels. 
+
+```powershell tab="PowerShell Core"
+# this examples searches the users in a directory by displayname
+$memberDisplayName = "A"
+# Group settings
+$logoPath = ".\pnpImage.png"
```

Comment:
Path with `\` breaks in PowerShell on macOS. Could you check if changing it to `/` would work on Windows? If it did, we could use it instead to allow using the script both on Windows and macOS.

Reply:
Hi,
it works also with /.

I am going to change it

Reply:
Awesome! Thank you for checking 👍